src/features/core/session/index.js

```diff
@@ -143,6 +140,11 @@ class UserSession {
     this.storage.setSessionInfo(this.sessionInfo);
   }
 
+  getSessionId() {
+    const session = this.getSessionInfo();
```

Comment:
This could lead to starting a new session if it's not already active. Is that side-effect fine?

Reply:
Addressed as per the discussion.